eclair-core/src/test/scala/fr/acinq/eclair/db/PgUtilsSpec.scala

```diff
@@ -99,6 +100,28 @@ class PgUtilsSpec extends TestKitBaseClass with AnyFunSuiteLike with Eventually
     pg.close()
   }
 
+  test("lock release utility method") {
+    val pg = EmbeddedPostgres.start()
+    implicit val ds: DataSource = pg.getPostgresDatabase
+
+    val lock1 = LeaseLock(UUID.randomUUID(), 2 minutes, 1 minute, LockFailureHandler.logAndThrow)
+    val lock2 = LeaseLock(UUID.randomUUID(), 2 minutes, 1 minute, LockFailureHandler.logAndThrow)
+
+    lock1.obtainExclusiveLock
```

Comment:
nit: it's really weird to call these without the parenthesis...

Reply:
Done with 61352e250.